operator/src/trustee.rs

```diff
@@ -151,21 +153,30 @@ pub async fn generate_kbs_configurations(
     let config_maps: Api<ConfigMap> = Api::namespaced(client, namespace);
 
     let kbs_config = include_str!("kbs-config.toml");
-    let data = BTreeMap::from([("kbs-config.toml".to_string(), kbs_config.to_string())]);
-    let config_map = ConfigMap {
-        metadata: kube::api::ObjectMeta {
-            name: Some(trustee.kbs_configuration.to_string()),
-            namespace: Some(namespace.to_string()),
+    let as_config = include_str!("as-config.json");
+    let rvps_config = include_str!("rvps-config.json");
+
+    for (filename, content, configmap) in [
+        ("kbs-config.toml", kbs_config, &trustee.kbs_configuration),
+        ("as-config.json", as_config, &trustee.as_configuration),
```

Comment:
This PR version has some of the merged changes restored, but not all of them (e.g. this older piece is still there). I don't think this PR needs to change most of what's above the `tests` module, just the `Deserialize` derive (I could be missing something that is required though).